<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>MainWindow</class>
 <widget class="QMainWindow" name="MainWindow">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>800</width>
    <height>600</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>MainWindow</string>
  </property>
  <widget class="QWidget" name="centralwidget">
   <widget class="QLabel" name="lblName">
    <property name="geometry">
     <rect>
      <x>60</x>
      <y>80</y>
      <width>67</width>
      <height>17</height>
     </rect>
    </property>
    <property name="text">
     <string>Name:</string>
    </property>
   </widget>
   <widget class="QLabel" name="lblAddress">
    <property name="geometry">
     <rect>
      <x>60</x>
      <y>140</y>
      <width>67</width>
      <height>17</height>
     </rect>
    </property>
    <property name="text">
     <string>Address:</string>
    </property>
   </widget>
   <widget class="QLabel" name="lblCountry">
    <property name="geometry">
     <rect>
      <x>60</x>
      <y>240</y>
      <width>67</width>
      <height>17</height>
     </rect>
    </property>
    <property name="text">
     <string>Country:</string>
    </property>
   </widget>
   <widget class="QTextEdit" name="txtName">
    <property name="geometry">
     <rect>
      <x>150</x>
      <y>70</y>
      <width>321</width>
      <height>31</height>
     </rect>
    </property>
   </widget>
   <widget class="QTextEdit" name="txtAddress">
    <property name="geometry">
     <rect>
      <x>150</x>
      <y>130</y>
      <width>321</width>
      <height>70</height>
     </rect>
    </property>
   </widget>
   <widget class="QListWidget" name="lstCountry">
    <property name="geometry">
     <rect>
      <x>150</x>
      <y>240</y>
      <width>321</width>
      <height>91</height>
     </rect>
    </property>
    <item>
     <property name="text">
      <string>India</string>
     </property>
    </item>
    <item>
     <property name="text">
      <string>Pakistan</string>
     </property>
    </item>
    <item>
     <property name="text">
      <string>Bangladesh</string>
     </property>
    </item>
    <item>
     <property name="text">
      <string>Nepal</string>
     </property>
    </item>
    <item>
     <property name="text">
      <string>China</string>
     </property>
    </item>
   </widget>
   <widget class="QPushButton" name="btnSubmit">
    <property name="geometry">
     <rect>
      <x>150</x>
      <y>370</y>
      <width>89</width>
      <height>25</height>
     </rect>
    </property>
    <property name="text">
     <string>Submit</string>
    </property>
   </widget>
   <widget class="QPushButton" name="btnReset">
    <property name="geometry">
     <rect>
      <x>310</x>
      <y>370</y>
      <width>89</width>
      <height>25</height>
     </rect>
    </property>
    <property name="text">
     <string>Reset</string>
    </property>
   </widget>
  </widget>
  <widget class="QMenuBar" name="menubar">
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>0</y>
     <width>800</width>
     <height>22</height>
    </rect>
   </property>
  </widget>
  <widget class="QStatusBar" name="statusbar"/>
 </widget>
 <resources/>
 <connections/>
</ui>
